.github/workflows/release.yml

```diff
@@ -0,0 +1,50 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+permissions:
+  packages: write
+  contents: read
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{github.actor}}
+          password: ${{secrets.GITHUB_TOKEN}}
+
+      - name: setup-pack
+        uses: buildpacks/github-actions/setup-pack@v5.5.1
+
+      - name: setup-tools
+        uses: buildpacks/github-actions/setup-tools@v5.5.1
+
+      - name: package
+        run: |
+          #!/usr/bin/env bash
+          set -euo pipefail
+
+          PACKAGE=${GITHUB_REPOSITORY}
+          VERSION=${{ github.event.release.tag_name }}
+          IMAGE="ghcr.io/${PACKAGE}:${VERSION}"
```

Comment:
Waar komt `PACKAGE` en `VERSION` vandaan? 🤔

Reply:
`GITHUB_REPOSITORY` is de repo naam, in dit geval "kadaster-labs/secured-sparql-endpoint". `github.event.release.tag_name` is de release tag versie, bv "0.1". Deze twee vormen samen de Docker image tag, dus "kadaster-labs/secured-sparql-endpoint:0.1"

Reply:
Heb ik daar zo hard overheen gelezen? 😳 🤔 🙈